tests/integration/test_payment_initiation.py

```diff
@@ -116,3 +116,42 @@ def test_all_payment_routes_with_iban():
     assert response['address'] is not None
 
     payments_after_recipient_creation(client, recipient_id)
+
+
+def test_payment_create_with_options():
```

Comment:
It looks like there isn't a test for the case where no options are provided? It may be a good idea to test that as well, to ensure that there isn't a regression

Reply:
There is a helper function `payments_after_recipient_creation` that creates payments without options and is called from a couple other tests.  I think we're good on ensuring no regression.  I can make a stand alone test if you feel that is helpful.